Mathlib/Analysis/BoxIntegral/Partition/Filter.lean

```diff
@@ -34,15 +34,15 @@ the corresponding integral, or in the proofs of its properties. We equip
 
 The structure `BoxIntegral.IntegrationParams` has 3 boolean fields with the following meaning:
 
-* `bRiemann`: the value `true` means that the filter corresponds to a Riemann-style integral, i.e.
+* `bRiemann`: the value `True` means that the filter corresponds to a Riemann-style integral, i.e.
```

Comment:
I wasn't careful with checking `True`/`true` and `False`/`false`, sorry. These ones should be lower case.

Could you check the PRs that I merged and revert any of changes that are terms of `Bool`?

Reply:
You're right... I just learned about this today. I've checked, only #11948 and this PR exhibit this issue. Hopefully, I can fix these tomorrow.

Reply:
Actually, this was easier than expected: I filed #11994 for the corrections.